scripts/bootstrap.sh

```diff
@@ -197,3 +196,13 @@ if [ "${CF_INSTANCE_INDEX:-''}" == "0" ] && [ -z "${SKIP_DRUPAL_BOOTSTRAP:-}" ];
 else
     echo "Bootstrap skipping Drupal CIM because: Instance=${CF_INSTANCE_INDEX:-''} Skip=${SKIP_DRUPAL_BOOTSTRAP:-''}"
 fi
+
+echo "Adding the USPS credentials..."
+if [[ ${USPS_USERID:-"unset"} != "unset" ]] && [[ ${USPS_PASSWORD:-"unset"} != "unset" ]]; then
+    echo "const USPS_USERID = '${USPS_USERID}';" > ./web/themes/custom/usagov/scripts/usps-credentials.js
+    echo "const USPS_PASSWORD = '${USPS_USERID}';" >> ./web/themes/custom/usagov/scripts/usps-credentials.js
```

Comment:
Bug here! It should be `${USPS_PASSWORD}` 
(I found this by inspecting my usps-credentials.js file, not by looking here; probably would have missed it.)